index.html

```diff
@@ -513,29 +513,140 @@ <h3>Assert Claim</h3>
+          
+          To check authenticity, the issuer's verification methods, such as
+          a public key, must be discoverable from the credential itself. It
+          is expected that the issuer identifier either is
+          cryptographically verifiable itself or that it can be used to
+          reliably discover the cryptographic material necessary to confirm
+          that the content of a credential has not been changed since
+          issuance. It is understood that verification depends on the
+          verifier's acceptance of the robustness of the cryptographic
+          mechanisms used for testing authenticity and for discovering
+          verification methods. An untrusted verification method cannot be
+          relied upon for verification.
+
+          To check the currency of a credential, verifiers must be able to
+          inquire about the current status of a credential without
+          revealing to the issuer 
+          <ol type="a">
+             <li>Who is requesting status</li>
+             <li>Which credential is being checked</li>
+             <li>The subject of the credential being checked</li> 
+          </ol>
+          This status check allows issuers to revoke, suspend, or otherwise
+          moderate the status of the credential.
+
+          Similarly, if a discovery mechanism is used to retrieve
+          verification methods for a given issuer, verifiers must be able
+          to do so without revealing unnecessary information to the issuer.
+
+          In short, it must be possible to fully verify a credential
+          without leaking usage data to the issuer or anyone else. This
+          avoids the so-called "phone home" problem.          
+          </dd>
+        <dt>Motivations</dt>
+        <dd>
+          In many environments (such as order processing) information such
+          as a payer's address, citizenship, or age need to be
+          automatically verified in order to complete the transaction. 
+          
+          It MUST be possible to verify in an automated fashion.
+        </dd>
+        <dt>Needs</dt>
+        <dd>
+          <uref>F.2</uref>, <uref>C.1</uref>, <uref>E.2</uref>, <uref>R.1
+          </uref>,
+          <uref>F.5</uref>, <uref>H.2</uref>, <uref>C.6</uref>
+        </dd>
+      </dl>
+    </section>
+    <section>
+      <h3>Validate Claim</h3>
+      <dl class="dl-horizontal">
+        <dt>Requirement</dt>
+        <dd>
+          It MUST be possible for a <a>verifier</a> to validate that the a
+          given claim is appropriate for particular use. That is, given the
+          credential is authentic and current, the verifier must then
+          interpret the claims in the context of their own 'business rules'
+          about which parties are acceptable authorities and which
+          cryptographic mechanisms are acceptable for different situations.
+          <ol>
+            <li>Is the issuer someone we know?</li>
```

Comment:
I think you're getting at "is the issuer a trustworthy entity" which may be more useful to state than simple knowledge

Reply:
We're generally trying to avoid buzzwords like "trust", "truth", "fact", etc., which can only be evaluated as/by opinion, but which are often read and interpreted as global and empirical absolutes.

Reply:
Yes, "trust" and "truth" are better avoided if we can state the technical implications without using the terms.